include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -579,8 +590,8 @@ __pattern_transform_scan_single_group(_ExecutionPolicy&& __exec, _InRng&& __in_r
                                       _BinaryOperation __binary_op, _Inclusive)
 {
     using _CustomName = typename std::decay_t<_ExecutionPolicy>::kernel_name;
```

Comment:
```suggestion
    using _CustomName = std::decay_t<_ExecutionPolicy>::kernel_name;
```

Reply:
The `typename` is needed not for the `decay_t`, but for the nested type inside `_ExecutionPolicy`. There is a compiler warning without it.

Reply:
Got it